Homebrew/BerserkerHybrid.part

```diff
@@ -73,13 +73,19 @@ On the edge of civilization, far beyond the comfortable life within cities and t
 </RulesElement>
 	<RulesElement name="Heartland" type="Class Feature" internal-id="ID_FMP_CLASS_FEATURE_4127" source="Heroes of the Feywild">
 		<Category>ID_CUSTOM_HYBRID_BERZERKER_BY_NUPTUP,1</Category>
-		<Flavor> Berserkers live on the edge of civilization, where they are closer to the land and more shaped by the environment than other warriors are. You have cultivated certain abilities that are based on your experiences in your heartland's environment. </Flavor>
+		<Flavor> Berserkers live on the edge of civilization, where they are closer to the land and more shaped by the environment than other warriors are. You have cultivated certain abilities that are based on your experiences in your heartland's environment. 
+
+Note: Hybrid Berserkers who choose this option are granted proficiency with Leather Armor, Hide Armor and Light Shields. </Flavor>
```

Comment:
I don't like putting important info in the Flavor field.

Reply:
Where else should it be put? For the warden hybrid talent it bakes it right into the description of the talent, but with Heartland it is a talent that grants a selection, so you can't really add it to each heartland selection separately so I don't know where else to place it.

Reply:
Eh, good enough for now.  I might fix it later